blackbox/docs/sql/administration/ingestion_rules.txt

```diff
@@ -0,0 +1,92 @@
+.. highlight:: psql
+.. _administration-ingestion-rules:
+
+===============
+Ingestion Rules
+===============
+
+In CrateDB, ingestion rules define where and how the incoming data of a
+specific ingestion should be routed and stored.
+
+In this section, we will learn about creating and deleting ingestion rules,
+listing the existing ingestion rules and finally the required privileges for
+the ``CREATE`` and ``DROP INGEST RULE`` statements, when the
+:ref:`administration_user_management` feature is enabled.
+
+Creating and Deleting Ingestion Rules
+.....................................
+
+Suppose we have installed an ingestion implementation for an ``mqtt`` broker,
+and we would like to filter temperature data streams by topic and store them to
+``temp_data.raw``::
+
+    cr> CREATE INGEST RULE temperature
```

Comment:
I assume the examples here are not executable? Is there any plan to change that?